bool is_attack
uint16 start_distance
uint16 end_distance
float32 radius
uint16 number_of_points
uint8 frequency